server/queue_priv.php

```diff
@@ -3,12 +3,89 @@
 	require_once "$root/server/config.php";
 	$conn = $DB_CONN;
```

Comment:
This is unused in the file, you can remove it

Reply:
When I remove those lines, the functions no longer work. I tried using just the server/config.php and that still didn't work.

Reply:
Sorry, I totally understand what is happening here now. So when you do this $conn = $DB_CONN, you're taking the global variable $DB_CONN and putting it into another global variable called $conn. And then you access this with that `$GLOBALS['conn']`. I think this is sort of redundant. I think what you should do instead is remove the `$conn = $DB_CONN`, and then do `$GLOBALS['DB_CONN']` instead. 

You will also see the structure in our codebase of 
```
GLOBAL $DB_CONN;
$DB_CONN->prepare(....);
```
Both the $GLOBALS associative array variable and GLOBAL keyword do the same thing (to my knowledge), so there is no difference, it's just a preference thing. Feel free to use whatever construct you like. 

But I do think you should not do this whole $conn = $DB_CONN thing. I can explain more about the PHP scoping in person when you have time.